include/sycl/algorithm/count_if.hpp

```diff
@@ -41,7 +41,7 @@
 namespace sycl {
 namespace impl {
 
-#if 0
+#ifdef __COMPUTECPP__
```

Comment:
This might not be `__COMPUTECPP__`, but at least Codeplay can change it if they want to try/use the other version.